src/Compilers/Server/VBCSCompilerTests/CompilerServerApiTest.cs

```diff
@@ -133,203 +107,6 @@ private static Task<T> FromException<T>(Exception ex)
             return source.Task;
         }
 
-        [Fact]
```

Comment:
Note: I did not just delete a bunch of tests here. Most of these tests were moved to `VBCSCompilerServerTests` or `ClientConnectionHandlerTests`. In some cases though I deleted tests that were duplicates of other tests we currently have.

Reply:
As a part of this move to new files I also converted all of these tests to be unconditional `[Fact]` tests where before many were conditioned to windows or windows desktop only.